packages/scripts.nix

```
@@ -207,6 +207,34 @@
     '';
   };
 
+  kubectl-wait-coordinator = writeShellApplication {
+    name = "kubectl-wait-coordinator";
+    runtimeInputs = with pkgs; [ kubectl ];
+    text = ''
+      namespace=$1
+      name="coordinator-0"
+
+      echo "Waiting for $name.$namespace to become ready" >&2
+
+      timeout=180
+
+      interval=4
+      while [ $timeout -gt 0 ]; do
```

```suggestion
      while [[ $timeout -gt 0 ]]; do
```
nit: prefer [[ for consistency.

---

It iis currently consistent with `kubectl-wait-ready` above. Let's fix this in a follow-up.